public/Miscellaneous/printYoutube.js

```diff
@@ -2,9 +2,9 @@ if (!window["youtubeIFrame"]) {
 	window["youtubeIframe"] = true;
 
 	window.addEventListener("load", () => {
-		
+
 		let candidates = $("iframe");
-		for (let i = 0; i < candidates.length; i++) {
```

Comment:
I think that this line is still necessary, as there may be multiple iframes on the page

Reply:
Ok. Just added that back in, not sure how it got deleted. Thanks for catching that